modules/000-common/images/distroless/file-whitelist

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
```

Comment:
I prefer not to use the term `whitelist`. Don't you think that it's better to name it `file-filter`?

Reply:
Fixed.